GillianCore/engine/Abstraction/Unifier.ml

```diff
@@ -1442,14 +1458,16 @@ module Make
                     in
                     make_resource_fail ())))
 
-  and unify_up' (parent_ids : L.ReportId.t list ref) (s_states : search_state')
-      : up_u_res =
+  and unify_up'
+      ?(is_post = false)
```

Comment:
```suggestion
      ~is_post
```

Reply:
There's no need to make this argument optional, it's already optional in the main function